source/documentation/how-to/hosted-engine.html.md

```diff
@@ -240,7 +240,7 @@ To boot from different media, e.g. a rescue CD:
 
 ## **More info**
 
-Additional information is available in the feature page [Features/Self_Hosted_Engine](Features/Self_Hosted_Engine)
+Additional information is available on the [Self Hosted Engine](/develop/release-management/features/engine/self-hosted-engine/) feature page.
 
```

Comment:
Are the new link still valid? I think you already moved the feature pages from engine to sla

Reply:
@JohnMarksRH ?

Reply:
@doron-fediuck @sandrobonazzola Sandro is right. It needs to be updated. I will deal with it next week.